docs/tutorial/airbyte_config_persistence.md

```diff
@@ -0,0 +1,524 @@
+
+Otherwise, please complete the different steps until you reach the Airbyte Dashboard page.
+
+After a few seconds, the UI should be ready to go at http://localhost:8000/
+
+## Notes about running this tutorial on Mac OS vs Linux
+
+Note that Docker for Mac is not a real Docker host, now it actually runs a virtual machine behind the scenes and hides it from you to make things simpler. 
+Simpler, unless you want to dig deeper... just like our current use case where we want to inspect the content of internal Docker volumes...
+
+Here are some related links as references on accessing Docker Volumes:
+- on Mac OS [Using Docker containers in 2019](https://stackoverflow.com/a/55648186)
+- official doc [Use Volume](https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes)
+
+From these discussions, we will be using on Mac OS either:
+1. any docker container/image to browse the virtual filesystem by mounting the volume in order to access them, for example with [busybox](https://hub.docker.com/_/busybox)
+2. or extract files from the volume by copying them onto the host with [Docker cp](https://docs.docker.com/engine/reference/commandline/cp/)
+
+However as a side remark on Linux, accessing to named Docker Volume can be easier since you simply need to:
+    
+    docker volume inspect <volume_name>
+    
+Then look at the "Mountpoint" value, this is where the volume is actually stored in the host filesystem and you can directly retrieve files directly from that folder.
+
+Back to this tutorial, commands shown below should work for both Mac OS and Linux !
+
+## Export Initial Setup
+
+Now let's first make a backup of the configuration state of your Airbyte instance by running the following commands.
+
+
+```bash
+TUTORIAL_DIR=$(pwd)/build/persitence-tutorial
+mkdir -p $TUTORIAL_DIR/my-setup
+
+docker cp airbyte-server:/data $TUTORIAL_DIR/my-setup
+```
+
+## Configure some Exchange Rate source and File destination
+
+Head back to http://localhost:8000/ and add more connectors. 
+Here is an example of configuration from an API source:
+
+![airbyte_config_persistence_api_source](./airbyte_config_persistence_1.png)
+
+and a local file destination:
+
+![airbyte_config_persistence_local_file](./airbyte_config_persistence_2.png)
+
+## Run a Sync job
+
+- once the source and destination are created
+- the catalog and frequency can be configured
+- then run the "Sync Now" button
+- finally inspect logs in the UI
+
+![airbyte_config_persistence_ui_logs](./airbyte_config_persistence_3.png)
+
+## Exploring Logs folders
+
+We can read from the lines reported in the logs the working directory that is being used to run the synchronization process from.
+
+As an example in the previous run, it is being ran in `/tmp/workspace/5/0` and we notice the different docker commands being used internally are starting with:
```

Comment:
the path to the logs is at the top of job history (currently only in dev but will be released to prod today), so that might make this explanation a little easier.